README.rst

```diff
@@ -80,3 +80,24 @@ To browse Thoth's logs during or after the adviser run:
 .. code-block:: console
 
   thamos log
+
+
+Security
+========
+
+One of the options that can be set when getting advice is setting the :code:`recommendation_type`.  On Thoth's adviser
+adviser this indicates different steps that should be included when scoring packages.  By setting 
+:code:`recommendation_type == security` adviser will include steps such as checking for CVE and scoring based on Thoth's
```

Comment:
To be more accurate s/steps/pipeline units/g - here and above. Pipeline units of any time can be added based on the recommendation type configuration option.